src/Evolution/DiscontinuousGalerkin/DgElementArray.hpp

```diff
@@ -14,21 +14,111 @@
 #include "Domain/InitialElementIds.hpp"
 #include "Domain/OptionTags.hpp"
 #include "Domain/Tags.hpp"
+#include "Evolution/Protocols.hpp"
+#include "Evolution/Tags.hpp"
+#include "IO/Importers/VolumeDataReader.hpp"
+#include "IO/Importers/VolumeDataReaderActions.hpp"
 #include "Parallel/ConstGlobalCache.hpp"
 #include "Parallel/Info.hpp"
 #include "Parallel/ParallelComponentHelpers.hpp"
+#include "ParallelAlgorithms/Actions/SetData.hpp"
+#include "Utilities/ProtocolHelpers.hpp"
 #include "Utilities/TMPL.hpp"
 #include "Utilities/TaggedTuple.hpp"
 
+/*!
+ * \brief Don't try to import initial data from a data file
+ *
+ * \see `DgElementArray`
+ */
+struct ImportNoInitialData {};
```

Comment:
This ImportNoInitialData struct seems odd, or at least oddly named. Not sure what a good alternative is. Maybe a better comment explaining when you use this struct.

Reply:
What do you think is unclear? I guess since I wrote the code I'm not sure what confused someone who reads it for the first time. These structs are only used as template parameters to `DgElementArray` to choose whether or not to import numeric initial data, and to configure the numeric initial data to import. `ImportNoInitialData` is the default.